File: BT_A1/upd_datos_csv_inst.php
<?php
$archivo = 'datos_inst.csv';
$save_location = 'C:/AppServ/www/powermeter/BT_A1/datos_inst.csv';
$url = 'http://panel.powermeter.com.ar/descargar/directa/inst/56ae1c10-059b-4764-abec-f7bdc5e56603/';
$destino = "index.php";
// Realizar la descarga y guardar el archivo en la ubicación especificada  
$response = file_get_contents($url);
if ($response !== false) {
    if (file_put_contents($save_location, $response) !== false) {
        $msg2 = "Archivo CSV descargado y guardado correctamente";
    } else {
        $msg2 = "Error al guardar el archivo";
    }
} else {
    $msg2 = "Error al descargar el archivo";
}
?>

<script>
    // Mostrar una alerta
    alert("<?php echo $msg2; ?>");
    
    // Redireccionar a index.php después de aceptar la alerta
    window.location.href = 'index.php';
</script>

File: BT_A1/old/upd_datos_csv_inst.py
import os
import requests
import time

url = 'http://panel.powermeter.com.ar/descargar/directa/inst/56ae1c10-059b-4764-abec-f7bdc5e56603/'
filename = 'datos_inst.csv'
save_location = 'C:/AppServ/www/mediciones/BT_A1/'
nombre_archivo = 'datos_inst.csv'
print()
print(f"url: {url}")
print()
time.sleep(1)
print(f"filename: {filename}")
print()
time.sleep(1)
print(f"save_location: {save_location}")
print()
# Realizar la petición GET
respuesta = requests.get(url)
# Guardar el archivo descargado
with open(nombre_archivo, 'wb') as archivo:
    archivo.write(respuesta.content)
print("finalizado")
#input("Presione cualquier tecla para salir")

